content/go/concepts/math-functions/terms/min/min.md

```diff
@@ -0,0 +1,72 @@
+---
+Title: 'Min()'
+Description: 'Returns the minimum value of two specified numbers.'
+Subjects:
+  - 'Code Foundations'
+  - 'Computer Science'
+Tags:
+  - 'Numbers'
+  - 'Arithmetic'
+  - 'Functions'
+CatalogContent:
+  - 'learn-go'
+  - 'paths/computer-science'
+---
+
+The **`Min()`** function returns the minimum value of two specified numbers.
+
+## Syntax
+
+```pseudo
+import "math"
+
+result := math.Min(num1, num2)
+```
+
+`math.Min()` returns the minimum value from `num1` and `num2`.
+
+## Example
```

Comment:
I noticed that these examples are the same as in the max entry. It makes sense to want to simplify and use what you've created before, but preferably all examples are original.

Reply:
What should i do?

Reply:
Should I change the examples?

Reply:
@Manikanta528 Yes, go ahead and change them. Then tag us so we can merge this.

Reply:
@SSwiniarski yes, updated the examples.